modules/howtos/pages/distributed-acid-transactions-from-the-sdk.adoc

```diff
@@ -104,9 +100,7 @@ From a transaction context you may get a document:
 include::example$transactions-example.php[tag=get,indent=0]
 ----
 
-// TODO check PHP relevance of following
-`get` will cause the transaction to fail with `TransactionFailed` (after rolling back any changes, of course).
-It is provided as a convenience method so the developer does not have to check the `Optional` if the document must exist for the transaction to succeed.
+`get` will cause the transaction to fail with `TransactionOperationFailedException` (after rolling back any changes, of course).
```

Comment:
TransactionFailedException

Reply:
That class doesn't appear to exist, pending clarification.